### PR TITLE
docs: update Korean Readme

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**

### CODE REVIEW

**Comment by Jaehyuk-Lee** [comment]
What does ~~remaining greased lightning time~~ mean in [readme](https://github.com/quisquous/cactbot/blob/main/README.md)? I don't know what exactly happened to Monk in v5.4. Is it removed?

**Comment by Echoring** [comment]
Oh sorry, this readme is written before it actually deleted and have not been updated.

**Comment by Jaehyuk-Lee** [comment]
okay, thanks 😄

File: README.md [hunk 1]
```diff
@@ -440,8 +440,8 @@ In the screenshot below the phases are named B1, B2, B3.  These autogenerate fro
 
 ![xephero screenshot](screenshots/xephero.png)
 
-The [rdmty](ui/dps/rdmty) dps meter is based on the same dps meter for miniparse, and updated
-for Stormblood jobs and recolored to match [fflogs](http://fflogs.com).
```
Comment (Jaehyuk-Lee) [inline README.md:444]: I removed Stormblood jobs part, it confuses whether the overlay is not compatible with Shadowbringers or not.

File: README.md [hunk 2]
```diff
@@ -440,8 +440,8 @@ In the screenshot below the phases are named B1, B2, B3.  These autogenerate fro
...
+The [rdmty](ui/dps/rdmty) dps meter is based on the same dps meter for miniparse, and
+recolored to match [fflogs](http://fflogs.com).
```
Comment (Jaehyuk-Lee) [inline README.md:444]: I removed Stormblood jobs part, it confuses whether the overlay is not compatible with Shadowbringers or not.

**Review by panicstevenson — APPROVED** [review]

**Pull request merged**